### PR TITLE
Add Get Banned Users Endpoint

### PR DESCRIPTION
**Pull request by Chronophylos**

### CODE REVIEW

File: moderation.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package helix
+
+// Ban ...
+// ExpiresAt must be parsed manually since an empty string means perma ban
+type Ban struct {
+	UserID    string `json:"user_id"`
+	UserName  string `json:"user_name"`
+	ExpiresAt string `json:"expires_at"`
```
Comment (nicklaw5) [inline moderation.go]: We have support for unmarshalling timestamps to `time.Time` values. Would you mind changing this type from `string` to `time.Time`? See the Stream struct for an example: https://github.com/nicklaw5/helix/blob/c9cfaa2ec39b467599a43368176732453ef91a2c/streams.go#L15

Comment (Chronophylos) [reply moderation.go]: I'd do that but if a user is permanently banned `expires_at` will be an empty string.

Comment (nicklaw5) [reply moderation.go]: Ah k. I was not aware of that.

Comment (nicklaw5) [reply moderation.go]: In that case let's use the custom `Time` struct that I introduced for this specific scenario:
https://github.com/nicklaw5/helix/blob/c9cfaa2ec39b467599a43368176732453ef91a2c/time.go#L16-L37

**Comment by nicklaw5** [comment]
Looks good! Appreciate the contribution :smiley:

**Pull request merged**